src/kirby/components/floating-action-button/floating-action-button.component.ts

```diff
@@ -9,4 +9,5 @@ export class FloatingActionButtonComponent {
   // TODO: showShadow should become an "elevation" enum in the future;
   @Input() showShadow?: boolean = true;
   @Input() disabled?: boolean = false;
+  @Input() size?: 'sm' | 'md' | 'lg' = 'sm';
```

Comment:
We normally use the short versions: 's', 'm' and 'l'

Reply:
not according to src/kirby/directives/size/size.directive.ts

Reply:
@unnamjolner We are in the process of aligning this across components (to all use 2-letter abbrevation)